Jenkinsfile

```diff
@@ -12,7 +12,7 @@ pipeline {
     stages {
         stage('prepare') {
             steps {
-             withCredentials([string(credentialsId: "api.rh-idev.openshift.com", variable: 'TOKEN')]) {
+             withCredentials([string(credentialsId: "${env.IGNITE_ACCOUNT_CREDENTIAL}", variable: 'TOKEN')]) {
```

Comment:
Is this change intentional? I guess `IGNITE_ACCOUNT_CREDENTIAL` and `--server=https://api.rh-idev.openshift.com` won't work very well.

Reply:
Yeah, not really sure why this slipped in. But this was a fix from @iocanel upstream so I wonder why it disappears here as a diff.